src/components/HomePage.js

```diff
@@ -0,0 +1,37 @@
+import { useSelector } from 'react-redux';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Header from './Header';
+
+const HomePage = () => {
```

Comment:
- [ ] Your Homepage is missing the descriptive banner section as contained in the original design. Kindly add it. 
- [ ] Also ensure the homepage contains information describing the data your app is showcasing.
- [ ] Kindly ensure these changes are reflected on the live demo

  Given design | Your implementation
  :---:|:---:
  ![debas-origin-home](https://user-images.githubusercontent.com/36542986/152330062-3c37a697-f5dc-4c52-bbb5-218dae9ac1b0.jpg) | ![debas-own-home](https://user-images.githubusercontent.com/36542986/152329793-b06779dd-9fc5-4ac7-be8c-9c8deac050e5.jpg)

Reply:
Good job!